apps/src/templates/lessonOverview/LessonOverview.jsx

```diff
@@ -178,7 +188,10 @@ class LessonOverview extends Component {
             {lesson.preparation && (
               <div>
                 <h2>{i18n.preparation()}</h2>
-                <SafeMarkdown markdown={lesson.preparation} />
+                <EnhancedSafeMarkdown
+                  markdown={lesson.preparation}
+                  expandableImages={true}
```

Comment:
I think you can just do

```suggestion
                  expandableImages
```

here and elsewhere